eventlog/el/main.go

```diff
@@ -168,97 +155,71 @@ func main() {
 }
 
 func create(c *cli.Context) error {
-	fn := c.String("roster")
-	if fn == "" {
-		return errors.New("-roster flag is required")
+	if c.Bool("keys") {
+		kp := key.NewKeyPair(cothority.Suite)
+		fmt.Println("Private: ", kp.Private)
+		fmt.Println(" Public: ", kp.Public)
+		return nil
 	}
 
-	in, err := os.Open(fn)
-	if err != nil {
-		return fmt.Errorf("Could not open roster %v: %v", fn, err)
-	}
-	r, err := readRoster(in)
-	if err != nil {
-		return err
-	}
+	return errors.New("not implemented")
+}
 
-	n := c.String("name")
-	cfg, err := doCreate(n, r, 5*time.Second)
-	if err != nil {
-		return err
-	}
-	fmt.Printf("Created event log chain with ID %x.\n", cfg.ID)
-	return err
+func show(c *cli.Context) error {
+	return errors.New("not implemented")
+	// long := c.Bool("long")
```

Comment:
I would remove the commented out code. It doesn't look like something you can't write (better) in 10 minutes...